.circleci/release.yaml

```diff
@@ -0,0 +1,33 @@
+{{- range .Values.versions }}
+apiVersion: release.giantswarm.io/v1alpha1
+kind: Release
+metadata:
+  annotations:
+    giantswarm.io/docs: https://pkg.go.dev/github.com/giantswarm/apiextensions/pkg/apis/release/v1alpha1?tab=doc#Release
```

Comment:
This should point to our new docs website once it's live.

Reply:
Do we have an issue tracking this or something?

Reply:
I guess it's this one https://github.com/giantswarm/giantswarm/issues/8804